src/integration-tests/model-in-image/README

```diff
@@ -0,0 +1,120 @@
+Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+SUMMARY:
+
+  This is a stand alone test for running the mii-sample in
+  'kubernetes/samples/scripts/create-weblogic-domain/model-in-image'.
+
+IMPORTANT NOTES:
+
+  (1) This test isn't currently integrated with the test
+      suite framework. Instructions for doing so are below.
+
+  (2) This test assumes Kubernetes is running locally 
+      on a single machine. (It builds/uses local images
+      without pushing them.)
+
+-------------------
+To run stand alone:
+-------------------
+
+  1) Optionally customize the following:
```

Comment:
Is this the first file user will look at (or the landing page is at the documentation)?  I am just wondering how the user knows where things occured

Reply:
No. This is a test. It's not meant for users.    See the files in the sample directory for the sample.

Read the pull summary for details.